hex-number/hex-number.py

```diff
@@ -0,0 +1,102 @@
+class Node:
```

Comment:
Your Pull Request title: HEXNUMBER-1021 Added hex-number.py and test-hex-number.py

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']